Install manpages uncompressed. It's a waste of time to compress them here
because MacPorts will just uncompress them and recompress them (to ensure
they're compressed with maximum compression).
--- buildtools/manpage.mk.orig	2024-12-22 00:38:06.000000000 -0600
+++ buildtools/manpage.mk	2024-12-22 23:13:41.000000000 -0600
@@ -1,3 +1,11 @@
+# srcdir.mk defines SRCDIR .
+include srcdir.mk
+BUILDDIR = $(CURDIR)
+SUBDIR = 
+VPATH=.:$(SRCDIR)
+
+include $(BUILDDIR)/config.mk
+
 # Make Unix man pages from Netpbm HTML user manual
 # GNU make version 3.81 or newer recommended.
 # Tested with GNU make version 3.80.
@@ -171,9 +178,9 @@
 .PHONY : installman
 installman: report
 	set -x
-	for f in $(wildcard $(MAN1)); do if [ -f $$f ]; then gzip <$$f >$(MANDIR)/man1/$$f.gz; fi; done
-	for f in $(wildcard $(MAN3)); do if [ -f $$f ]; then gzip <$$f >$(MANDIR)/man3/$$f.gz; fi; done
-	for f in $(wildcard $(MAN5)); do if [ -f $$f ]; then gzip <$$f >$(MANDIR)/man5/$$f.gz; fi; done
+	for f in $(wildcard $(MAN1)); do [ -f $$f ] && $(INSTALL) -c -m $(INSTALL_PERM_MAN) $$f $(MANDIR)/man1; done
+	for f in $(wildcard $(MAN3)); do [ -f $$f ] && $(INSTALL) -c -m $(INSTALL_PERM_MAN) $$f $(MANDIR)/man3; done
+	for f in $(wildcard $(MAN5)); do [ -f $$f ] && $(INSTALL) -c -m $(INSTALL_PERM_MAN) $$f $(MANDIR)/man5; done
 
 
 # This will uninstall the man pages.
